tests/test-cases.js

```diff
@@ -246,14 +317,14 @@ var CORE_PARSER_TESTS = [
 				"code": "InvalidQuotes",
 				"message": "Trailing quote on quoted field is malformed",
 				"row": 0,
-				"index": 3
+				"index": 2
```

Comment:
Why changing the index?

Reply:
as mentioned in the explanation, the index should represent the beginning of the quote or column and not inside the quoted string.